open_spiel/games/universal_poker/README.md

```diff
@@ -2,9 +2,8 @@
 
 This has been contributed by dennisjay in November 2019 (See
 https://github.com/deepmind/open_spiel/pull/97), and is available as an optional
-dependency. See the
-[https://github.com/deepmind/open_spiel/blob/master/docs/install.md](install.md)
-for documentation and `open_spiel/scripts/global_variables.sh` to enable this.
+dependency. See the [install.md](/docs/install.md) for documentation
```

Comment:
This renders well on GitHub, not sure if it is a good practice or relative link is better? Or the absolute link. At any rate the label and the URL are switched in the current file.